docs/source/user/transformations/aggregation-sum.ipynb

```diff
@@ -0,0 +1,419 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Aggregation: Sum\n",
+    "\n",
+    "This notebook is a deep-dive on transformations for computing the sum with bounded stability.\n",
+    "\n",
+    "---\n",
+    "Any constructors that have not completed the proof-writing and vetting process may still be accessed if you opt-in to \"contrib\".\n",
+    "Please contact us if you are interested in proof-writing. Thank you!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from opendp.mod import enable_features\n",
+    "enable_features(\"contrib\")\n",
+    "\n",
+    "from opendp.transformations import *"
+   ]
+  },
+  {
```

Comment:
Visually, this could use some kind of separation above this cell. (Other notebooks happen to have a heading after the contrib disclaimer.)